tests/CascadeTest.php

```diff
@@ -70,8 +71,8 @@ public function testGetLoggerWithInheritance()
     public function testGetLoggerWithInheritanceAndNonAlphanumericParentNames()
     {
         Cascade::setInheritOnUndefined(true);
-        $parent = Cascade::getLogger('logger_A');
-        $child = Cascade::getLogger('logger_A.child_A');
+        $parent = Cascade::getLogger('logger_Aé');
```

Comment:
I wanted to make sure the logging system wouldn't break upon reading other characters allowed in PHP class names. Technically, the correct regex would be `'[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*'` but it's probably out of scope to worry about that.